lisa/sut_orchestrator/azure/features.py

```diff
@@ -13,6 +13,7 @@
 from PIL import Image, UnidentifiedImageError
 
 from lisa import features, schema, search_space
+from lisa.features.nvme import NvmeOptionSettings
```

Comment:
Put all features in `__init__.py` of features folder. So they can be imported from `lisa.features` together.

Reply:
updated